test/k8s-integration/main.go

```diff
@@ -324,18 +334,21 @@ func runTestsWithConfig(testDir, testFocus, testConfigArg string) error {
 	artifactsDir, _ := os.LookupEnv("ARTIFACTS")
 	reportArg := fmt.Sprintf("-report-dir=%s", artifactsDir)
 
-	kubetestArgs := fmt.Sprintf("--ginkgo.focus=%s --ginkgo.skip=%s %s %s",
+	testArgs := fmt.Sprintf("--ginkgo.focus=%s --ginkgo.skip=%s %s %s",
 		testFocus,
 		"\\[Disruptive\\]|\\[Serial\\]|\\[Feature:.+\\]",
 		testConfigArg,
 		reportArg)
 
-	cmd := exec.Command("kubetest",
+	kubeTestArgs := []string{
 		"--test",
 		"--ginkgo-parallel",
-		fmt.Sprintf("--test_args=%s", kubetestArgs),
-	)
-	err = runCommand("Running Tests", cmd)
+		fmt.Sprintf("--test_args=%s", testArgs),
+	}
+
+	kubeTestArgs = append(kubeTestArgs, cloudProviderArgs...)
+
+	err = runCommand("Running Tests", exec.Command("kubetest", kubeTestArgs...))
```

Comment:
did you verify this all works, kubetest args can get pretty crazy sometimes

Reply:
I verified it locally, and it passed 31/32 tests. I suspect its because locally it runs on many more "nodes" or kubetest workers, so more tests run concurrently. It was saying across 25 nodes.